src/lib/db.ts

```
@@ -0,0 +1,21 @@
+  connect_timeout: 10,
+});
+
+export const db = drizzle(client, { schema });
+
+// Test connection function
+export async function testConnection() {
+  try {
+    const result = await db.execute('SELECT 1 as test');
+    return result;
+  } catch (error) {
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)